setup.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash 
+
+conda create -n xnat_downloader_env python=2.7
```

Comment:
to make this more automated, add a `-y` at the end of the statement (this will just download the packages without asking the user for conformation.
ex: `conda create -n xnat_downloader_env python=2.7 -y`

see my comment below on editing this line.